.github/workflows/automerge.yml

```diff
@@ -0,0 +1,60 @@
+name: automerge
+on:
+  schedule:
```

Comment:
Problem: Scheduled runs only happen against the default branch, not against PRs.

Reply:
oh, that's a boomer... So the only way would be for a dedicated action to go through open PRs and see if any need to be triggered to be reran or do you see some other way?

Reply:
Do you mean "merged" instead of "reran"?

Aside from that, yes, that's pretty much the only way.